### DISCUSSION TITLE
ultimate sd upscale in comfyui. keep getting error

### DISCUSSION THREAD

**Discussion by jimfor2011** [ssitu/ComfyUI_UltimateSDUpscale#65] [General]
Cannot import C:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_UltimateSDUpscale module for custom nodes: cannot import name 'devices' from 'modules' (C:\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-reactor-node\modules\__init__.py)

Any suggestions to fix. installed comfyui manager to install and also through cmd in custom nodes.

**Comment by ssitu** [comment]
For some reason there is a conflict between this and reactor node, which I don't have myself. Try disabling one.
